### PR TITLE
Adds OffsetArray support

### LINKED ISSUE #79: OffsetArray support
Currently,

```julia
using OffsetArrays, ImageBinarization
Xo = OffsetArray(rand(10, 10), OffsetArrays.Origin(0))
binarize(Xo, Otsu()) # oops
```

Background: common `warp` operations(e.g., `imrotate`) in ImageTransformations returns an `OffsetArray`.

### PR DESCRIPTION
**Pull request by zygmuntszpak**
Resolves #79

### CODE REVIEW

File: test/algorithms/single_histogram_threshold.jl [hunk 1]
```diff
@@ -98,4 +98,17 @@
...
+    @testset "Offset Arrays" begin
+        # Verify that the algorithms give equivalent results on OffsetArrays
+        img = imresize(testimage("cameraman"); ratio=0.25)
+        offset_img = OffsetArray(img, OffsetArrays.Origin(0))
+        for (fname, f) in threshold_methods
+            img₀₁ = binarize(img, f)
+            offset_img₀₁ = binarize(offset_img, f)
+            for i in zip(img₀₁, offset_img₀₁)
+                @test first(i) == last(i)
+            end
```
Comment (johnnychen94) [inline test/algorithms/single_histogram_threshold.jl]: maybe

```suggestion
            @test axes(offset_img₀₁) == axes(offset_img)
            @test img₀₁ == OffsetArrays.no_offset_view(offset_img₀₁)
```

**Review by johnnychen94 — APPROVED** [review]
Just one small suggestion to the test

The appveyor can be removed from the repository settings page: https://github.com/zygmuntszpak/ImageBinarization.jl/settings/installations

File: test/algorithms/single_histogram_threshold.jl (hunk 1 above)
Comment (zygmuntszpak) [reply test/algorithms/single_histogram_threshold.jl]: Great! Didn't know about `OffsetArrays.no_offset_view(offset_img₀₁)`!

Comment (zygmuntszpak) [reply test/algorithms/single_histogram_threshold.jl]: I removed the Appveyor webhook. I don't have Appveyor listed here either:  https://github.com/zygmuntszpak/ImageBinarization.jl/settings/installations

**Comment by johnnychen94** [comment]
~Because this one bumps the version, I'll tag a new release.~

Edit:

Never mind... I'll leave it to you in case you have more commits comming.

Just leave a comment `@JuliaRegistrator register`  in the commit you want to tag a release, e.g., https://github.com/zygmuntszpak/ImageBinarization.jl/commit/d5b3ba6891c490df640fdb472477b977978c2b88

**Pull request merged**